release-notes/major10/10.4.0/valtimo-backend-libraries.md

```diff
@@ -11,10 +11,15 @@ which can be used in message throw events as an expression.
 More detail [here](/using-valtimo/process/correlating-messages.md).
 
 
-* **New feature2**
+* **Object management**
```

Comment:
The release notes are only meant to describe what new things have been added in this release. 
The release notes don't describe how implementation can use these new things. That belongs under 'using-valtimo'.

For example: 'A new 'search' module has been added that manages searching through objectApi-objects. See [this] page for more information'

Also, you can add your documentation to the branch 'feature/object-management' and make a PR out of that